domain-ee/ee-ep-merge-app/src/python_src/util/custom_enum.py

```diff
@@ -0,0 +1,17 @@
+from enum import Enum
+
+
+class StrEnum(str, Enum):
+    """
+    StrEnum subclasses enum.Enum and overrides `enum.auto()` to have values equal to their names.
+    Enums inheriting from this class that set values using `enum.auto()` will have values equal to their names.
+    """
+
+    # noinspection PyMethodParameters
+    def _generate_next_value_(name, start, count, last_values) -> str:  # type: ignore
+        """
+        Uses the name as the automatic value, rather than an integer
+
+        See https://docs.python.org/3/library/enum.html#using-automatic-values for reference
```

Comment:
Cute!

Yeah, this is a neat trick. This URL doesn't have the fragment for me though; https://docs.python.org/3/howto/enum.html#using-automatic-values is the one for me. 🤷